airflow/hooks/mysql_hook.py

```diff
@@ -19,36 +19,39 @@
 
 
 class MySqlHook(DbApiHook):
-    '''
+    """
     Interact with MySQL.
 
     You can specify charset in the extra field of your connection
     as ``{"charset": "utf8"}``. Also you can choose cursor as
     ``{"cursor": "SSCursor"}``. Refer to the MySQLdb.cursors for more details.
-    '''
+    """
 
     conn_name_attr = 'mysql_conn_id'
     default_conn_name = 'mysql_default'
     supports_autocommit = True
 
+    def __init__(self, *args, **kwargs):
+        super(MySqlHook, self).__init__(*args, **kwargs)
```

Comment:
I would suggest adding schema as an explicit keyword here and not passing it to the super call. However I think `DbApiHook` does some ugly args/kwargs parsing that makes that difficult. Instead, perhaps use `kwargs.pop('schema', None)` to keep the kwargs clean.